final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import logging as log
+import sys
+import datetime
+import arguments
+import printers
+import rss_get_items
+import converting
+import DatabaseEmulation
+from copy import deepcopy
+
+now = datetime.datetime.now()
+
+""" Set basic configs for logging """
+stdoutHandler = log.StreamHandler(sys.stdout)
+fileHandler = log.FileHandler("logging.log", "a", encoding="utf-8")
+log.basicConfig(format='%(levelname)-8s [%(asctime)s] %(message)s',
+                level=log.DEBUG,
+                handlers=[fileHandler])
+
+
+def print_verbose() -> None:
+    log.info('try to read log file')
+    with open('logging.log', 'r', encoding="utf-8") as f:
+        print(f.read())
+
+
+if __name__ == '__main__':
+
+    log.info("Start script")
+
+    database = DatabaseEmulation.DatabaseEmulation(
```

Comment:
This class should support context manager protocol and use it.

Reply:
Didn't catch this! Could you explain in more detail?